### PR TITLE
Apply black style to T* files, version 19.10b0.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to test_T* files, 4 files. it should be fine to merge.
test_TCoffee_tool.py
test_TogoWS.py
test_translate.py
test_TreeConstruction.py

Skipped file test_Tutorial.py as is failing, I'll check what can be done.

### CODE REVIEW

File: Tests/test_TCoffee_tool.py [hunk 1]
```diff
@@ -64,16 +64,21 @@ def test_TCoffee_fasta(self):
...
         cmdline.infile = self.infile1
         cmdline.outfile = self.outfile3
         cmdline.output = "pir_aln"
-        self.assertEqual(str(cmdline), t_coffee_exe + " -output pir_aln "
-                         "-infile Fasta/fa01 -outfile Fasta/tc_out.pir -quiet")
+        self.assertEqual(
+            str(cmdline),
+            t_coffee_exe + " -output pir_aln "
+            "-infile Fasta/fa01 -outfile Fasta/tc_out.pir -quiet",
+        )
```
Comment (peterjc) [inline Tests/test_TCoffee_tool.py]: Merge the two string literals into one?

File: Tests/test_TCoffee_tool.py [hunk 2]
```diff
@@ -93,25 +100,37 @@ def test_TCoffee_clustalw(self):
...
         cmdline.type = "protein"
-        self.assertEqual(str(cmdline), t_coffee_exe + " -output clustalw_aln "
-                         "-infile Fasta/fa01 -outfile Fasta/tc_out.aln "
-                         "-type protein -outorder input -gapopen -2 -gapext -5")
+        self.assertEqual(
+            str(cmdline),
+            t_coffee_exe + " -output clustalw_aln "
+            "-infile Fasta/fa01 -outfile Fasta/tc_out.aln "
+            "-type protein -outorder input -gapopen -2 -gapext -5",
+        )
```
Comment (peterjc) [inline Tests/test_TCoffee_tool.py]: Can probably save a line by changing where the long string is split?

File: Tests/test_TCoffee_tool.py [hunk 3]
```diff
@@ -121,15 +140,24 @@ def test_TCoffee_phylip(self):
...
+            infile=self.infile1,
+            outfile=self.outfile6,
+            quiet=True,
+            output="msf_aln",
+        )
+        self.assertEqual(
+            str(cmdline),
+            t_coffee_exe + " -output msf_aln "
+            "-infile Fasta/fa01 -outfile Fasta/tc_out.msf -quiet",
+        )
```
Comment (peterjc) [inline Tests/test_TCoffee_tool.py]: Merge the two string literals into one?

File: Tests/test_TCoffee_tool.py (hunk 1 above)
Comment (svalqui) [reply Tests/test_TCoffee_tool.py]: Done.

File: Tests/test_TCoffee_tool.py (hunk 2 above)
Comment (svalqui) [reply Tests/test_TCoffee_tool.py]: black splits on the addition, it didn't save a line but looks more standard.

File: Tests/test_TCoffee_tool.py (hunk 3 above)
Comment (svalqui) [reply Tests/test_TCoffee_tool.py]: Done, Thank you for the comments.

**Comment by peterjc** [comment]
Thanks again.

**Pull request merged**